x/concentrated-liquidity/lp_test.go

```diff
@@ -875,14 +885,18 @@ func (s *KeeperTestSuite) TestUpdatePosition() {
 			expectedError:  true,
 		},
 		"new position when calling update position - error because fee accumulator is not initialized": {
-			poolId:         1,
-			ownerIndex:     1, // using a different address makes this a new position
-			lowerTick:      DefaultLowerTick,
-			upperTick:      DefaultUpperTick,
-			joinTime:       DefaultJoinTime,
-			liquidityDelta: DefaultLiquidityAmt,
-			numPositions:   1,
-			expectedError:  true,
+			poolId:                    1,
+			ownerIndex:                1, // using a different address makes this a new position
+			lowerTick:                 DefaultLowerTick,
+			upperTick:                 DefaultUpperTick,
+			joinTime:                  DefaultJoinTime,
+			liquidityDelta:            DefaultLiquidityAmt,
+			numPositions:              1,
+			amount0Expected:           DefaultAmt0Expected,
+			amount1Expected:           DefaultAmt1Expected,
+			expectedPositionLiquidity: DefaultLiquidityAmt,
+			expectedTickLiquidity:     DefaultLiquidityAmt.Add(DefaultLiquidityAmt),
+			expectedPoolLiquidity:     DefaultLiquidityAmt.Add(DefaultLiquidityAmt),
```

Comment:
This test case still says error but we no longer expect an error here.

Reply:
renamed